### PR TITLE
New Updates

### PR DESCRIPTION
**Pull request by lavender-sys**
I have changed the PONG command to say PING. Also, it new says its prefix in the playing/status area before stating the game that was set. I will continue to update it.

### CODE REVIEW

File: cmd.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ exports.commands = {
...
   "pong": {
     process: function(bot, message) {
-      bot.sendMessage(message.channel, "Die in a hole.");
+      bot.sendMessage(message.channel, "PING");
     }
   },
   "setgame": {
     process: function(bot, msg, suffix) {
-      bot.setStatus('online', suffix);
+      bot.setStatus('online', Prefix: = Playing: suffix);
```
Comment (Dougley) [inline cmd.js]: This is incorrectly stringified, else it looks okay

Comment (lavender-sys) [inline cmd.js]: Okay. I'm guessing the error is that I forgot the "s (Duble Quotes) around "Prefix:= Playing: "
Fixing now.

Comment (lavender-sys) [inline cmd.js]: Fixed!

**Comment by DoNotSpamPls** [comment]
If nobody has problems with this PR, I will merge it in 24 hours!

File: cmd.js [hunk 2]
```diff
@@ -8,12 +8,12 @@ exports.commands = {
...
   "pong": {
     process: function(bot, message) {
-      bot.sendMessage(message.channel, "Die in a hole.");
+      bot.sendMessage(message.channel, "PING");
     }
   },
   "setgame": {
     process: function(bot, msg, suffix) {
-      bot.setStatus('online', suffix);
+      bot.setStatus('online', "Prefix: = Playing:" suffix);
```
Comment (Dougley) [inline cmd.js]: Again, the string usage is correct, but to add the `suffix` object is should state `+ suffix`

Comment (DoNotSpamPls) [inline cmd.js]: Did you fix everything?

**Comment by lavender-sys** [comment]
Soon I'll add a idle setgame command.

**Comment by Dougley** [comment]
All my complaints are gone, if nobody else has issues this is cleared to be merged

**Comment by lavender-sys** [comment]
It's all ready to merge.

**Pull request merged**